js/main.js

```diff
@@ -143,6 +172,12 @@ function viewSwapping(dataView) {
 
         allDataView[i].appendChild(profilePage(data.profile));
 
+      } else {
```

Comment:
This `else` block and the conditional it is paired with above do not need to be in the loop. Once you have swapped the visibility of all the views to the proper state, this conditional can be run just once compared to the `dataView` passed into the function 👍